### PR TITLE
fix #406, default executor jvm config in gear.conf is not effective.

### LINKED ISSUE #406: default executor jvm config in gear.conf is not effective
When setting executor JVM config in gear.conf, I observed the JVM config is not used in runtime.

### PR DESCRIPTION
**Pull request by clockfly**
[Review on Reviewable](https://reviewable.io/reviews/intel-hadoop/gearpump/407)

### CODE REVIEW

File: core/src/main/scala/org/apache/gearpump/util/Util.scala [hunk 1]
```diff
@@ -82,16 +82,16 @@ object Util {
 
     import Constants._
 
-    val appMasterVMArgs = Try(conf.getString(GEARPUMP_APPMASTER_ARGS).split(" ")).toOption
-    val executorVMArgs = Try(conf.getString(GEARPUMP_EXECUTOR_ARGS).split(" ")).toOption
+    val appMasterVMArgs = Try(conf.getString(GEARPUMP_APPMASTER_ARGS).split("\\s+").filter(!_.isEmpty)).toOption
```
Comment (manuzhang) [inline core/src/main/scala/org/apache/gearpump/util/Util.scala]: `filter(_.nonEmpty)` could be more concise

File: streaming/src/main/scala/org/apache/gearpump/streaming/appmaster/ExecutorManager.scala [hunk 2]
```diff
@@ -54,6 +54,7 @@ private[appmaster] class ExecutorManager (
   private var taskManager: ActorRef = null
   private var currentExecutorId = 0
   implicit val actorSystem = context.system
+  private var systemConfig = context.system.settings.config
```
Comment (manuzhang) [inline streaming/src/main/scala/org/apache/gearpump/streaming/appmaster/ExecutorManager.scala]: could this be a `val` ?

**Comment by clockfly** [comment]
Thanks, fixed.

**Comment by manuzhang** [comment]
+1

**Pull request merged**